launcher/helper.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * jsonutils.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+VCMI_LIB_NAMESPACE_BEGIN
```

Comment:
this is not part of the VCMI lib, no need to add this macro. And this is exactly the source of the iOS error.

Reply:
Thanks! :)